src/index.ts

```diff
@@ -135,6 +139,12 @@ export class BufferedChangeset implements IChangeset {
     }
   }
 
+  /**
+   * @property maybeUnwrapProxy
+   * @override
+   */
+  maybeUnwrapProxy = maybeUnwrapProxy;
```

Comment:
in ember-changeset, this will look like

```
  function unwrapProxy(o) {
    return isProxy(o) ? unwrapProxy(get(o, 'content')) : o;
  }

  function isProxy(o) {
    return !!(o && (o instanceof ObjectProxy || o instanceof ArrayProxy));
  }
```

Reply:
@andreyfel What do you think about this PR?  Would love your feedback!!